code/drasil-code/lib/Language/Drasil/Code/Imperative/Descriptions.hs

```diff
@@ -74,7 +74,7 @@ inputConstructorDesc = do
       icDesc True = "checking " ++ pAndS ++ " on the input"
       dl = defList g
   return $ "Initializes input object by " ++ stringList [ 
-    ifDesc ("get_input" `elem` dl),
+    ifDesc (showChsStr GetInput `elem` dl),
```

Comment:
So this now becomes a 'hack' at the String level. So the question is, what is being represented by `dl`, and could it be done otherwise?

Reply:
Is the `defList` here part of the [Text.Html](https://hackage.haskell.org/package/html-1.0.1.2/docs/src/Text-Html.html#defList) module?  [DrasilState.hs](https://github.com/JacquesCarette/Drasil/blob/92dddf7a545ba5029f99ad5c5eddcd8dad56a2d8/code/drasil-code/lib/Language/Drasil/Code/Imperative/DrasilState.hs) has this as `defList :: [Name]`, I'm not quite sure what is happening with `dl = defList g`